docs/source/main_classes/processors.mdx

```diff
@@ -65,12 +65,7 @@ Those processors are:
 Additionally, the following method can be used to load values from a data file and convert them to a list of
 [`~data.processors.utils.InputExample`].
 
-automethod,transformers.data.processors.glue.glue_convert_examples_to_features
-
-
-### Example usage
-
-An example using these processors is given in the [run_glue.py](https://github.com/huggingface/transformers/tree/master/examples/legacy/text-classification/run_glue.py) script.
+[[autodoc]] data.processors.glue.glue_convert_examples_to_features
```

Comment:
From my limited doc knowledge, `automethod` is from the old style. Not sure if this is the correct fix (did some pattern copy), the link to the staging docs seems broken .

Reply:
This is all deprecated code, but thanks for fixing the doc!